src/main.ts

```diff
@@ -73,6 +90,17 @@ export { type Board, BoardClient } from './components/board';
 export { default as cameraApi } from './gen/component/camera/v1/camera_pb';
 export { type Camera, type MimeType, CameraClient } from './components/camera';
 
+/**
+ * Raw Protobuf interfaces for a Camera component.
```

Comment:
Is this accurate for the `DataClient` type export?

Reply:
Nope! Haha, I fixed that :)